src/main/java/com/insightfullogic/honest_profiler/ports/sources/FileLogSource.java

```diff
@@ -71,8 +71,8 @@ public FileLogSource(final File file)
         this.file = file;
         try
         {
-            raf = new RandomAccessFile(file, "r");
-            channel = raf.getChannel();
+            this.raf = new RandomAccessFile(file, "r");
```

Comment:
You've got some auto formatting thing on which basically adds "this." everywhere. You may like it, but it makes for a very noisy diff :-( 
This diff Should be a 2 lines fix. There's no need for the incremental approach here either, given that it's a trivial change. Making it an incremental PR just delays it's acceptance.